src/server/network/protocol/protocolgame.cpp

```diff
@@ -5604,7 +5604,21 @@ void ProtocolGame::sendMarketDetail(uint16_t itemId, uint8_t tier) {
 	}
 
 	if (!oldProtocol) {
-		msg.add<uint16_t>(0x00); // Augment
+		if (it.augments.size() > 0) {
+			std::ostringstream augmentsString;
+			uint8_t count = 0;
+			for (auto &augment : it.augments) {
```

Comment:
```suggestion
			for (const auto &augment : it.augments) {
```

Reply:
I don't like either, I saw some aliases and I thought that was a pattern to follow, reverting.